atomicfu/src/wasmMain/kotlin/kotlinx/atomicfu/AtomicFU.kt

```diff
@@ -0,0 +1,178 @@
+/*
```

Comment:
Is it possible to simply set the Wasm's sources dir to the  JS source's dir instead of copying files? Or there's a downside of compiling code w/ `JsName` into Wasm?

Reply:
We could try to do that using intermediate MPP sourceset (but then we need to disable metadata tasks) or include shared directory onto two sourcets. In both cases JsName have to be typealiased in JS and stubbed in Wasm (expect/actual or not). There was not so much copy-pasted code so I decided for a while not to do that. Do you think it worth?

Reply:
It's worth reducing the amount of duplicated code.

I tried to simply set Wasm's sourceDir to Js's one and everything worked fine without:
> stubbed in Wasm (expect/actual or without).

Or you meant that it's already done?

Reply:
Its better NOT to mark Wasm declarations with JsName in this case. So for Wasm target this annotation should be stubbed. I can do it and you will see what I mean.